Main/lib/sql/Postgres/createVdiControlTables.sql

```diff
@@ -186,6 +278,16 @@ GRANT SELECT ON VDI_CONTROL_:VAR1.dataset_publication TO gus_r;
 GRANT SELECT ON VDI_CONTROL_:VAR1.dataset_hyperlink TO gus_r;
```

Comment:
`dataset_properties` refs still exist above and below this line (can't comment on lines that aren't included in the pr diff)